physics/MP/Ferrier_Aligo/mp_fer_hires.meta

```diff
@@ -55,7 +55,7 @@
   long_name = MPI communicator
   units = index
   dimensions = ()
-  type = integer
+  type = MPI_Comm
```

Comment:
I missed this change probably because this scheme is not used in ufs. I can update the type of mpicomm to MPI_Comm, but I have no way of testing if it works. How should we test this? Which model is using this scheme?

Reply:
This is fine as long as the scheme isn't assuming this field is an integer. 
I know "mpirank" and "mpiroot" are referenced as integer in schemes (e.g. if (mpirank .eq. mpiroot) then ), but not mpicomm as far as I can tell.

Reply:
I think so, too. As long as the only purpose of passing MPI_Comm to the scheme is to use it in MPI function calls that will work - since these MPI calls will be either against the f08 interface (which expect the MPI_COMM type) or the old interface(which expect the old integer). I think the same is true for the netcdf interface?